docs/gaps/gaps.tex

```diff
@@ -0,0 +1,286 @@
+\usepackage{graphics}
+\usepackage{placeins}
+\usepackage{booktabs} % nice rules (thick lines) for tables
+\usepackage{microtype} % improves typography for PDF
+\usepackage{xspace}
+\usepackage[hidelinks]{hyperref}
+\usepackage{xspace}
+\usepackage{hhline}
+
+\usepackage{tabularx}
+\newcolumntype{b}{X}
+\newcolumntype{s}{>{\hsize=.5\hsize}X}
+\newcolumntype{m}{>{\hsize=.75\hsize}X}
+
+\newcommand{\SN}{S$_N$}
+\renewcommand{\vec}[1]{\bm{#1}} %vector is bold italic
+\newcommand{\vd}{\bm{\cdot}} % slightly bold vector dot
+\newcommand{\grad}{\vec{\nabla}} % gradient
+\newcommand{\ud}{\mathop{}\!\mathrm{d}} % upright derivative symbol
+\newcommand{\Cyclus}{\textsc{Cyclus}\xspace}%
+\newcommand{\Cycamore}{\textsc{Cycamore}\xspace}%
+\graphicspath{ {images/} }
+\usepackage[affil-it]{authblk}
+\usepackage{tikz}
+\usetikzlibrary{positioning, arrows, decorations, shapes }
+\usepackage{cleveref}
+
+
+\usepackage[acronym,toc]{glossaries}
+\include{acros.tex}
+	
+\makeglossaries
+
+\title{Non-algorithmic Capability Gaps for \Cyclus and \Cycamore for transition analyses }
+\author{Jin Whan Bae, Kathryn Huff}
+\affil{Dept. of Nuclear, Plasma, and Radiological Engineering, University of Illinois at Urbana-Champaign
+		  Urbana, IL}
+\date{}                     %% if you don't need date to appear
+\setcounter{Maxaffil}{0}
+\renewcommand\Affilfont{\itshape\small}
+%%%%%%%%%%%%%%actual words%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%5
+
+
+\begin{document}
+
+\section{Introduction}
+As part of NEUP-FY16-10512 (PI: Dr. Kathryn Huff), a test-run of fuel cycle
+transition scenarios was done using \Cyclus. The purpose of this study
+is to identify the non-algorithmic capability gaps necessary for a complete simulation.
+The fuel cycle models are from the Idaho National Laboratory Nuclear Fuel Cycle Evaluation and Screening Report
+(Hereby E\&S Report) \cite{wigeland_nuclear_2014}. The transition scenarios begin with EG01 and transitions to EG23, EG24, EG29, EG30,
+separately. The descriptions of each fuel cycle is in \cref{tab:es}.
+
+
+\begin{table}[h]
+	\centering
+	\resizebox{\textwidth}{!}{
+		\begin{tabular}{|c|c|}
+			\hline
+			Fuel Cycle & Description \\ \hline
+			EG01 & Once-through using \gls{UOX} fuel in \glspl{LWR} \\ \hline
+			EG23 & Continuous recycle of U/Pu with new natural U fuel in fast critical reactors \\ \hline 
+			EG24 & Continuous recycle of U/TRU with new natural U fuel in fast critical reactors \\ \hline
+			EG29 & Continuous recycle of U/Pu with new natural U fuel in both fast and thermal critical reactors \\ \hline
+			EG30 & Continuous recycle of U/TRU with new natural U fuel in both fast and thermal critical reactors \\ \hline
+		\end{tabular}
+		}
+		\caption{Evaluation groups identified by the E\&S report.}
+		\label{tab:es}
+\end {table} 
+
+
+
+\section{Simulation Specifics}
+The transition scenarios in this study follows a common set of base-case parameters and assumptions,
```

Comment:
"The transition scenarios in this study follows" --> "The transition scenarios in this study follow"